### PR TITLE
Add `ERC*Pausable` warning for public pausing mechanism

### PR DESCRIPTION
**Pull request by ernestognw**
Mitigates #4006, although it doesn't solve, makes it clear that there are missing functions.

### CODE REVIEW

**Review by Amxx — DISMISSED** [review]

**Comment by frangio** [comment]
This needs to be applied in ERC721Pausable and ERC1155Pausable as well.

File: contracts/token/ERC20/extensions/ERC20Pausable.sol [hunk 1]
```diff
@@ -12,6 +12,12 @@ import "../../../security/Pausable.sol";
  * Useful for scenarios such as preventing trades until the end of an evaluation
  * period, or having an emergency switch for freezing all token transfers in the
  * event of a large bug.
+ *
+ * WARNING: Using it requires securely exposing and adding access control to both
+ * {Pausable-_pause} and {Pausable-_unpause} internal functions since {Pausable} is
+ * intended to be used through inheritance. This is done in order to allow both
+ * {AccessControl} and {Ownable} pause restriction mechanisms. Not doing so will
+ * make the contract unpausable.
```
Comment (frangio) [inline contracts/token/ERC20/extensions/ERC20Pausable.sol:20]: What do you think about:

```suggestion
 * IMPORTANT: This contract does not include public pause and unpause functions. In
 * addition to inheriting this contract, you must define both functions, invoking the
 * {Pausable-_pause} and {Pausable-_unpause} internal functions, with appropriate
 * access control, e.g. using {AccessControl} or {Ownable}. Not doing so will
 * make the contract unpausable.
```

**Comment by ernestognw** [comment]
> This needs to be applied in ERC721Pausable and ERC1155Pausable as well.

Thanks for the heads up. I added them with the same message proposed.

File: .changeset/new-ways-own.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+'openzeppelin-solidity': patch
+---
+
+`ERC*Pausable`: Add note regarding missing public pausing functionality
```
Comment (frangio) [inline .changeset/new-ways-own.md:5]:
```suggestion
`ERC20Pausable`, `ERC721Pausable`, `ERC1155Pausable`: Add note regarding missing public pausing functionality
```

**Review by frangio — DISMISSED** [review]
Looks good!

**Review by frangio — APPROVED** [review]

**Pull request merged**